src/collective/vdexvocabulary/treevocabulary.py

```diff
@@ -9,6 +9,13 @@
 
 import imsvdex.vdex
 import os
+import six
+
+try:
+    from Products.CMFPlone.utils import safe_text
+except ImportError:
+    # Plone 5.1
+    from Products.CMFPlone.utils import safe_unicode as safe_text
```

Comment:
See above about the dependency on Plone

Reply:
I replaced `safe_text` with `six.ensure_text`.